optuna/study.py

```diff
@@ -580,6 +583,7 @@ def create_study(
         study_name=None,  # type: Optional[str]
         direction='minimize',  # type: str
         load_if_exists=False,  # type: bool
+        force_garbage_collection=True,  # type: bool
```

Comment:
Please add `force_garbage_collection` to `load_study` in order to turn off GC when users resume studies.